src/sentry/sentry_apps/api/endpoints/sentry_apps_stats.py

```diff
@@ -23,7 +23,7 @@ class SentryAppsStatsEndpoint(SentryAppsBaseEndpoint):
     def get(self, request: Request) -> Response:
         sentry_apps = (
             SentryApp.objects.filter(installations__date_deleted=None)
-            .annotate(Count("installations"))
+            .annotate(installations__count=Count("installations"))
```

Comment:
django-stubs improved `annotate` support -- but doesn't support non-named arguments.  setting the name to the default name is the simplest fix